CHANGELOG.md

```diff
@@ -125,4 +125,14 @@ Released with 1.0.0-beta.37 code base.
 - fix: export bloom functions on the index.js
 - Prefer receipt status to code availability on contract deployment (#3298)
 
+## [1.2.6]
+
+### Added
+
+- ENS Görli testnet registry added 
+
+### Changed
+
+- ENS registry addresses updated
```

Comment:
The release notes should give a bit more context on what is happening here and why this is changed, eventually also linking to a reliable external source.

If the changes are done here in the in the release PR itself, there should be a self-reference to this PR #3353 at the end of the change items.